int64 start_x
int64 start_y
int64 end_x
int64 end_y
string color # color of cone, 'b' = blue, 'y' = yellow, 'r' = red
